### PR TITLE
Segment Info

### PR DESCRIPTION
**Pull request by mkong0216**
When hovering over a segment after route selection, info box displays: speed, id, segment id, and tile id. 
When hovering over a segment after region selection, info box displays: speed, osmlr id, and id. 

There's a lint-js error because I assign osmlr_id directly from properties.osmlr_id but lint-js wants it to be camel case. Should I change it?

### CODE REVIEW

**Comment by louh** [comment]
In this case it's just easier to leave it alone since `osmlr_id` is how you got it. You can [disable an eslint warning when it's getting in the way](https://eslint.org/docs/user-guide/migrating-from-jscs#disabling-rules-inline).

Single line disable:
```js
console.log(osmlr_id); // eslint-disable-line camelcase
```

Multi line disable:
```js
/* eslint-disable camelcase */
const { osmlr_id } = feature.properties;
console.log(osmlr_id);
/* eslint-enable camelcase */
```

File: src/app/route-info.js [hunk 1]
```diff
@@ -13,13 +13,14 @@ export function displayRegionInfo (selection) {
...
-  const { speed, osmlr_id, id } = selection.feature.properties
+  /* eslint-disable camelcase */
+  const { speed, osmlr_id, id } = selection.feature.properties 
   featureInfo.innerHTML =
     `<p> SPEED: ${speed ? speed.toFixed(2) : 0} kph <br/>
-         OSMLR_ID: ${osmlr_id} <br/>
+         OSMLR_ID: ${osmlr_id} <br/> 
          ID: ${id} <br/>
      </p>`
+  /* eslint-disable camelcase */
```
Comment (louh) [inline src/app/route-info.js]: this should be `/* eslint-enable ...` (to re-enable a temporarily disabled rule)

File: src/app/route-info.js [hunk 2]
```diff
@@ -13,13 +13,14 @@ export function displayRegionInfo (selection) {
   // create the feature info box
   if (!featureFlag) createFeatureInfo()
   setPosition(selection.pixel.x, selection.pixel.y)
-
-  const { speed, osmlr_id, id } = selection.feature.properties
+  /* eslint-disable camelcase */
+  const { speed, osmlr_id, id } = selection.feature.properties
```
Comment (louh) [inline src/app/route-info.js]: looks like there are trailing spaces introduced here that is now breaking the test

**Comment by mkong0216** [comment]
I just merged the changes from master and now I'm getting a bunch of lint-js errors from those changes. Should I fix those?

**Comment by mkong0216** [comment]
By the way, I pulled in changes from master and now when you point and click if there is no data for that specific week then the route just doesn't render. Is that what's supposed to happen now or is that a bug?

**Comment by louh** [comment]
Yep - that's a bug, but separate from this PR, so I'll note in issues. Thanks for catching it!

**Pull request merged**